microprofile-fault-tolerance/pom.xml

```diff
@@ -99,5 +99,130 @@
                 </plugins>
             </build>
         </profile>
+        <!-- Test against Bootable JAR -->
+        <profile>
+            <id>bootablejar.profile</id>
+            <properties>
+                <!-- TBD - remove once https://issues.redhat.com/browse/WFARQ-74 is done and wildfly-arquillian-3.0.0.Beta2 is out -->
+                <jboss.home>${project.build.directory}</jboss.home>
```

Comment:
We're using Beta2 already, see https://github.com/fabiobrz/eap-microprofile-test-suite/blob/bootable-jar/pom.xml#L42, hence I'd say this can be removed.